src/app/admin/createCollectionCentre/CreateCollectionCentreForm.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { CenterComponent, FormErrorText, StyledForm } from "@/components/Form/formStyling";
+import Button from "@mui/material/Button";
+import {
+    checkErrorOnSubmit,
+    Errors,
+    FormErrors,
+    setError,
+    setField,
+} from "@/components/Form/formFunctions";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faBuildingCircleArrowRight } from "@fortawesome/free-solid-svg-icons";
+import RefreshPageButton from "@/app/admin/RefreshPageButton";
+import { Schema } from "@/database_utils";
+import NameCard from "@/app/admin/createCollectionCentre/createCollectionCentreFormSections/NameCard";
+import AcronymCard from "@/app/admin/createCollectionCentre/createCollectionCentreFormSections/AcronymCard";
+import supabase from "@/supabaseClient";
+
+const initialFields: Schema["collection_centres"] = {
+    name: "",
+    acronym: "",
+};
+
+const initialFormErrors: FormErrors = {
+    name: Errors.initial,
+    acronym: Errors.initial,
+};
+
+const formSections = [NameCard, AcronymCard];
+
+const CreateCollectionCentreForm: React.FC<{}> = () => {
+    const [fields, setFields] = useState(initialFields);
+    const [formErrors, setFormErrors] = useState(initialFormErrors);
+
+    const fieldSetter = setField(setFields, fields);
+    const errorSetter = setError(setFormErrors, formErrors);
```

Comment:
Can we move this to after all state definition.

Reply:
Done